### ISSUE TITLE
Window does not center properly when used with jQuery 3.1.1

### ISSUE THREAD

**Issue by veselints** [telerik/kendo-ui-core#2882]
### Bug report
Window does not center properly when used with jQuery 3.1.1.

### Reproduction of the problem
- Run the following [Dojo](http://dojo.telerik.com/OcEyiY/6);
- Click the button to open the Window;
- Close the Window;
- Click the button to open the Window again - it is not centered but moved to the right;

### Expected/desired behavior
Window should be centered after calling the *center()* method

### Environment
* **jQuery version:** 3.1.1

**Comment by Vasimovic** [comment]
Looks centered to me, I used Chrome, Edge, and Firefox

**Comment by Iankodj** [comment]
The problem is caused due to a jQuery bug related to the returned calculated width and height (https://github.com/jquery/jquery/issues/3193), which is fixed with jQuery 3.2.0 and we encourage you to use it (or 3.2.2) instead of 3.1.1 version.

Possible workaround is to adjust the close animation and prevent transform animations like zoom: 

http://dojo.telerik.com/OcEyiY/7
```
...
    animation: {
        close: {
            effects: "fade:out",
            duration: 250
        }
    }
...
```


Or disable the close animation:

http://dojo.telerik.com/OcEyiY/8
```
...
    animation: {
        close: false
    }
...
```
